sdkit/models/model_loader/stable_diffusion/__init__.py

```diff
@@ -1,54 +1,81 @@
 import os
-from omegaconf import OmegaConf
+from pathlib import Path
+import tempfile
+from urllib.parse import urlparse
+
 from ldm.util import instantiate_from_config
-from transformers import logging as tr_logging
-from torch.nn.functional import silu
 import ldm.modules.attention
 import ldm.modules.diffusionmodules.model
-import tempfile
-from urllib.parse import urlparse
-from pathlib import Path
+from omegaconf import OmegaConf
+from torch.nn.functional import silu
+from transformers import logging as tr_logging
+
 
 from sdkit import Context
-from sdkit.utils import load_tensor_file, save_tensor_file, hash_file_quick, download_file, log
+from sdkit.utils import load_tensor_file, save_tensor_file, hash_file_quick,\
+     download_file, log
+
+tr_logging.set_verbosity_error()  # suppress unnecessary logging
 
-tr_logging.set_verbosity_error() # suppress unnecessary logging
 
-def load_model(context: Context, scan_model=True, check_for_config_with_same_name=True, **kwargs):
-    from . import optimizations
+def load_model(
```

Comment:
Not every function signature needs to be multi-line. This one is quite clear and readable in a single line, so we don't have to break it into multiple lines.